mixer/tools/adapterlinter/main.go

```diff
@@ -36,54 +38,79 @@ var invalidImportPaths = map[string]string{
 		"wants to see it.",
 }
 
+const (
+	dots = "/..."
+)
+
 func main() {
 	flag.Parse()
 	for _, r := range getReport(flag.Args()) {
-		reportErr(r)
+		// check if it skipped
+		if dat, err := ioutil.ReadFile(r.file); err == nil {
```

Comment:
Doesn't this increase the scope of nolint? Linter will start ignoring the whole file?

Reply:
It looks for the line number from the report and only skips the particular line-bound report.

Reply:
Ah, the lines[] lookup below. It is not easy to see that. Can you add a comment to clarify?

Reply:
Done